src/ol/style/iconstyle.js

```diff
@@ -426,7 +426,7 @@ ol.style.IconImage_ = function(image, src, size, crossOrigin, imageState) {
    * @type {boolean}
    */
   this.tainting_ = false;
-
+  if (this.imageState_ == ol.style.ImageState.LOADED)this.determineTainting_();
```

Comment:
If you could fix it would be great! I need to practice my rebasing and updating of existing pulls elsewhere so I don't mess anything up. ;-)